libs/repositories/src/UsdRepository/UsdRepositoryCache.ts

```diff
@@ -1,131 +0,0 @@
-import { inject, injectable } from 'inversify';
-import {
-  PricePoint,
-  PriceStrategy,
-  UsdRepository,
-  deserializePricePoints,
-  serializePricePoints,
-} from './UsdRepository';
-import { SupportedChainId } from '@cowprotocol/shared';
-import ms from 'ms';
-import { CacheRepository } from '../CacheRepository/CacheRepository';
-import { getCacheKey, PartialCacheKey } from '../utils/cache';
-
-const NULL_VALUE = 'null';
-
-@injectable()
-export class UsdRepositoryCache implements UsdRepository {
-  private baseCacheKey: PartialCacheKey[];
-
-  constructor(
-    private proxy: UsdRepository,
-    private cache: CacheRepository,
-    private cacheName: string,
-    private cacheTimeValueSeconds: number,
-    private cacheTimeNullSeconds: number
-  ) {
-    this.baseCacheKey = ['repos', this.cacheName];
-  }
-
-  async getUsdPrice(
-    chainId: SupportedChainId,
-    tokenAddress: string
-  ): Promise<number | null> {
-    // Get price from cache
-    const key = getCacheKey(
-      ...this.baseCacheKey,
-      'usd-price',
-      chainId,
-      tokenAddress
-    );
-    const usdPriceCached = await this.getValueFromCache({
-      key,
-      convertFn: parseFloat,
```

Comment:
I don't see `convertFn` parameter in the new implementation of the repository, do I miss it?

Reply:
Good catch! refactored to consider this

Reply:
I was testing again and I couldn't integrate the proxy changes to the inversify framework, so I reverted this changes